pkg/tenant/pb/tenant.proto

```
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package tenant.v1;
```

Buf (1.61.0)

Files with package "tenant.v1" must be within a directory "tenant/v1" relative to root but were in directory "pkg/tenant/pb".

(PACKAGE_DIRECTORY_MATCH)